app/atoms/ln-patternlab/_ln-m-widget-contact.scss

```diff
@@ -0,0 +1,26 @@
+$_phone_prefix: 'T';
```

Comment:
Ok, we can leave them as examples.
However, the location of the files is wrong - these aren't atoms.
And do we really need the ln-patternlab folder? Anything from patternlab will have the ln- prefix already.